kratos/solving_strategies/strategies/residualbased_newton_raphson_strategy.h

```diff
@@ -638,10 +633,6 @@ class ResidualBasedNewtonRaphsonStrategy
         if (mInitializeWasPerformed == false)
             Initialize();
```

Comment:
```suggestion
```
To be consistent, this should be removed as well. However, I'd wait to another PR

Reply:
good point, if we agree with this, I will continue with that later